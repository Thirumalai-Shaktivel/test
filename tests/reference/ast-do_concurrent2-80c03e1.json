{
    "basename": "ast-do_concurrent2-80c03e1",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/do_concurrent2.f90",
    "infile_hash": "afe8788676d4b7be4b765236db5d9c60aa267325a08b714b35ba7eb6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-do_concurrent2-80c03e1.stdout",
    "stdout_hash": "cdc27826d3a1beda9a2fc9b91823194788e319583b02cbc97438ce15",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}