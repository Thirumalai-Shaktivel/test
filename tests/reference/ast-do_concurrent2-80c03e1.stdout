(TranslationUnit [(Subroutine do_concurrent2 [(a) (b)] () [] [] [(Declaration (AttrType TypeReal [] ()) [(AttrIntent InOut)] [(a [(() () DimensionExpr)] ()) (b [(() () DimensionExpr)] ())]) (Declaration (AttrType TypeInteger [] ()) [] [(i [] ())]) (Declaration (AttrType TypeReal [] ()) [] [(x [] ())])] [(= x (Real "1.0")) (DoConcurrentLoop [(ConcurrentControl i 1 10 ())] () [(ConcurrentShared [i]) (ConcurrentLocal [x]) (ConcurrentDefault)] [(If (> (FuncCallOrArray a [] [(() i ())] []) 0) [(= x (FuncCallOrArray sqrt [] [(() (FuncCallOrArray a [] [(() i ())] []) ())] [])) (= (FuncCallOrArray a [] [(() i ())] []) (- (FuncCallOrArray a [] [(() i ())] []) (** x 2)))] []) (= (FuncCallOrArray b [] [(() i ())] []) (- (FuncCallOrArray b [] [(() i ())] []) (FuncCallOrArray a [] [(() i ())] [])))]) (Print () [x])] [])])
